config/development.json

```diff
@@ -93,5 +93,6 @@
 		"ALLOWED_API_KEYS": "thisisasupersecureapikeythatisabsolutelysave"
 	},
 	"TRAINING_URL": "https://lernen.dbildungscloud.de",
-	"FEATURE_ROOMS_ENABLED": true
+	"FEATURE_ROOMS_ENABLED": true,
```

Comment:
I guess FEATURE_ROOMS_ENABLED should be by default true, and so here removed

Reply:
Thank you for the feedback! I will change it 👍